src/telemetry/debian/files

```diff
@@ -0,0 +1 @@
+telemetry_0.1_amd64.deb net extra
```

Comment:
this is autogenerated file, please remove.

Reply:
Ok